app/src/main/kotlin/com/hippo/ehviewer/ui/settings/AboutScreen.kt

```diff
@@ -89,49 +83,56 @@ fun AboutScreen(navigator: DestinationsNavigator) {
     ) { paddingValues ->
         Column(modifier = Modifier.nestedScroll(scrollBehavior.nestedScrollConnection).verticalScroll(rememberScrollState()).padding(paddingValues)) {
             Preference(
-                title = stringResource(id = R.string.settings_about_declaration),
-                summary = stringResource(id = R.string.settings_about_declaration_summary),
+                title = settingsAboutDeclaration,
+                summary = settingsAboutDeclarationSummary,
             )
             HtmlPreference(
-                title = stringResource(id = R.string.settings_about_author),
-                summary = author(),
+                title = settingsAboutAuthor,
+                summary = buildAnnotatedString {
+                    withStyle(SpanStyle(textDecoration = TextDecoration.LineThrough)) {
+                        appendLine("Hippo")
+                        appendLine("NekoInverter")
+                        appendLine("飛鳥澪")
+                    }
+                    append("Foolbar")
```

Comment:
Move to `EnTranslations`.

Reply:
Yes, but so do categories. I think it's better to gather strings (even untranslatable) together.